gremlin-console/src/main/static/licenses/asm

```diff
@@ -0,0 +1,27 @@
+ASM: a very small and fast Java bytecode manipulation framework
```

Comment:
already in the groovy jar. i dont' think we need to include it.

Reply:
Sure, I wasn't aware that inclusion in groovy was sufficient. I will remove ASM from the licenses.